Classes/ResolverInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace Wwwision\GraphQL;
+
+interface ResolverInterface
```

Comment:
I tried to add a short doc comment for every class/interface. I think that makes it much easier to comprehend the code